extensions/spotify-player/package.json

```diff
@@ -324,6 +324,14 @@
           "required": true
         }
       ]
+    },
+    {
+      "name": "queue",
+      "title": "Queue",
+      "subtitle": "Spotify",
+      "description": "See your queue and remove any tracks or episodes from it.",
```

Comment:
Is this description accurate? Can you remove tracks from it?

Reply:
Oops, good cache